docling/document_converter.py

```diff
@@ -274,6 +328,27 @@ def convert_all(
         max_file_size: int = sys.maxsize,
         page_range: PageRange = DEFAULT_PAGE_RANGE,
     ) -> Iterator[ConversionResult]:
+        """
+        Converts multiple documents fetched from file paths, URLs, or DocumentStreams.
+
+        Args:
+            source: Source of input documents given as an iterable of
+                file paths, URLs, or DocumentStreams.
+            headers: Optional headers given as a (single) dictionary
+                of string key-value pairs, in case of URL input source.
+            raises_on_error: Whether to raise an error on the
+                first conversion failure.
+            max_num_pages: Maximum number of pages to convert.
```

Comment:
`max_num_pages` description should be clarified as:
```
Maximum number of pages accepted per document. Documents exceeding this number will be skipped.
```

`page_range` should be clarified as:

```
Range of pages to convert in each document.
```